src/classes/TrafficControl.mjs

```diff
@@ -154,11 +154,7 @@ class AuthenticatedUserEntity extends TrafficEntity {
    */
   get isAdmin () {
     const user = this.#user
-    return Permissions.groups.find((group) => {
-      return group.isAdministrator && user.groups.find((uGroup) => {
-        return uGroup.id === group.id
-      })
-    })
+    return user.permissions.includes('users.write')
```

Comment:
Maybe we could add rate limits to the group itself. The limiter could pick the highest number from the user's groups with the default authenticated limit as a fallback. This would resolve the need to use `users.write` as some form of pseudo-administrator flag.